### ISSUE TITLE
Quest: Proving Grounds - Missing Enemies in Wildpaw Cavern

### ISSUE THREAD

**Issue by Badgermilk0** [ZhengPeiRu21/mod-individual-progression#403]
The quest Proving Grounds for Alterac Valley cannot be completed.
https://www.wowhead.com/classic/quest=7161/proving-grounds

The cave is empty and the Frostwolf Banner is nowhere to be found:
https://www.wowhead.com/classic/item=17850/frostwolf-banner

See youtube clip in the second link on how the mobs should be in there. :)

Edit*
My level is 52 if the bracket makes a difference

**Comment by StraysFromPath** [comment]
You can manually spawn the npcs/object with .npc add and .gobject add but you're 100% right the areas should be populated.

This issue is also present for alliance cave/quest of the same name.

**Comment by Badgermilk0** [comment]
@Grimfeather , is this something you can take a look at?

**Comment by Grimfeather** [comment]
Yes, I could maybe do this. But isn't this quest also available in WotLK?
If so, this should get fixed by AzerothCore.

I can probably copy a lot of the information I need from vMangos. Anyways, I'll have a look.

**Comment by Badgermilk0** [comment]
You are correct, I have not looked if this issue exists in "stock" AzerothCore, did not consider that.

**Comment by Grimfeather** [comment]
![Image1](https://github.com/user-attachments/assets/5e5513f2-d103-4052-8330-c280c9c3abdb)

I checked the alliance and horde caves. Both do have the banner, so the quest can be completed.
The yetis are indeed missing. I'll see if I can make a sql file to add them to the caves.
it shouldn't be part of this module. but it's nice to sorta fix it.

**Comment by Badgermilk0** [comment]
This is embarrasing.. the flag is indeed there now next to a Wildpaw Brute, it was not yesterday when I checked. =(

Then the issue is not a blocker, rather if mobs should be added again.

**Comment by Grimfeather** [comment]
@Badgermilk0 I been working on this a bit today. 

I used the spawn locations from the vMangos project. I placed the yetis, harpies and gnolls in the caves, but the TBC and WotLK versions of AV are different. So I'm still working on phasing out these creatures for TBC and WotLK and placing the correct creatures in the caves for those expansions. I just don't know yet which creatures to place in the caves for TBC and WotLK.
I also need their locations in the caves.

Anyone have this information? maybe from live servers, or Warmane perhaps. else I'll have to search Youtube for videos.
Anyways, making progress. It's coming.
